package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "gofaster-poc",
+  "title": "Web Compat",
+  "description": "Urgent post-release fixes for web compatibility.",
+
+  "license": "MPL-2.0",
+  "repository": "https://github.com/mozilla/webcompat-addon",
+  "version": "0.0.1",
+
+  "scripts": {
+    "jake": "./node_modules/.bin/jake"
+  },
+
+  "devDependencies": {
+    "eslint": "^3.4.0",
+    "eslint-plugin-json": "^1.2.0",
+    "eslint-plugin-mocha": "^4.5.1",
+    "eslint-plugin-mozilla": "0.0.3",
+    "eslint-plugin-promise": "^2.0.1",
+    "jake": "^8.0.12"
```

Comment:
If we need node.js version > 5, I think we can add  "check-node-version" to devDependencies and run "check-node-version --node '>= 5.0.0'" before jake export-mc. Or we can make the node check in jake. Both are good for me.